chia/wallet/wallet_transaction_store.py

```diff
@@ -3,20 +3,42 @@
 import dataclasses
 import logging
 import time
+from enum import IntEnum
 from typing import Dict, List, Optional, Tuple
 
 from chia.types.blockchain_format.sized_bytes import bytes32
 from chia.types.mempool_inclusion_status import MempoolInclusionStatus
 from chia.util.db_wrapper import DBWrapper2
 from chia.util.errors import Err
 from chia.util.ints import uint8, uint32
+from chia.util.streamable import Streamable, streamable
 from chia.wallet.transaction_record import TransactionRecord
 from chia.wallet.transaction_sorting import SortKey
 from chia.wallet.util.transaction_type import TransactionType
 
 log = logging.getLogger(__name__)
 
 
+class FilterMode(IntEnum):
```

Comment:
You should imo reuse `chia.wallet.wallet_coin_store.FilterMode` and not just duplicate it. Not sure if it adds some circular dependency somewhere, but if so you should probably move it to some other place, maybe `chia.wallet.wallet_types` or so.